Survey.xcodeproj/project.pbxproj

```diff
@@ -1560,10 +1590,13 @@
 				1C94DA1228996C8100ED0984 /* SError.swift in Sources */,
 				1CB18675288E534400372D50 /* ActivityTracker.swift in Sources */,
 				1CD4E37828A0CF6200EA1E8B /* StoreTokenUseCase.swift in Sources */,
+				1C044B9D28ACD9A5001FF098 /* HomeUseCase.swift in Sources */,
 				1C94DA1E289BBC9B00ED0984 /* ErrorTextModifier.swift in Sources */,
 				1CEE602828A4ABE100748C63 /* BodyTextModifier.swift in Sources */,
 				1C94DA1A289BA04800ED0984 /* ErrorResponse.swift in Sources */,
 				1CEE603028A4B77D00748C63 /* HeaderHomeView.swift in Sources */,
+				1C044B9F28ACDF58001FF098 /* AuthenInterceptor.swift in Sources */,
```

Comment:
I suggest using word in the dictionary for naming `AuthenticationIntercepter` `Authen` has no meaning and could create ambiguity.